script/typecheck

```diff
@@ -7,10 +7,10 @@ BOLD=$(tput bold)
 END=$(tput sgr0)
 
 echo "${BOLD}jsonlint${END}"
-jsonlint package.json > /dev/null && echo "Good"
+jsonlint package.json > /dev/null
```

Comment:
Given `set -e`, you could just put `echo "Good"` on the following line to get the intended effect.